components/BackgroundView/BackgroundView.jsx

```diff
@@ -0,0 +1,45 @@
+import { View } from "react-native";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+import { theme } from "../../utils/theme";
+
+
+/**
+ * BackgroundView. Background View which covers the whole screen and contains
+ * children in the safe area.
+ *
+ * @param {object} props
+ * @param {JSX.Element[]} [props.children] children elements to render inside
+ * @returns {JSX.Element}
+ */
+function BackgroundView(props) {
+
+  const {
+    children = [],
+  } = props;
+
+  const insets = useSafeAreaInsets();
+
+  return(
+    <View
+      style={{
```

Comment:
I'd recommend moving this into a react native stylesheet and into a seperate BackgroundView.styles.js file